### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [caarmen/poet-assistant#178]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Poet Assistant](https://play.google.com/store/apps/details?id=ca.rmen.android.poetassistant&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Poet Assistant app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201119_095212](https://user-images.githubusercontent.com/25429624/99645035-fc154480-2a4e-11eb-8ace-dfa363eacfe7.png)
